test/src/main/java/org/apache/accumulo/test/functional/AccumuloClientIT.java

```diff
@@ -149,6 +149,7 @@ public void testAccumuloClientBuilder() throws Exception {
     props.put(ClientProperty.INSTANCE_ZOOKEEPERS.getKey(), zookeepers);
     props.put(ClientProperty.AUTH_PRINCIPAL.getKey(), user1);
     props.put(ClientProperty.INSTANCE_ZOOKEEPERS_TIMEOUT.getKey(), "22s");
+    props.put(ClientProperty.SCAN_SERVER_DISPATCHER_OPTS_PREFIX + "enabled", "true");
```

Comment:
What is the purpose of this?

Reply:
This was from an early modification. I'll remove it.

Reply:
Removed in 7521ae6